File: pregunta 1/estructurado/uno.scala
object Hola {
    def fibonacci(n: Int)={
        var a= 1
        var b = 0
        var i=0
        while(i<n){
            println(b)
            b = a+b
            a = b-a
            i = i+1
        }
    }
    def main(args:Array[String]):Unit = {
        println("Ingrese un numero:")
        val n = scala.io.StdIn.readInt()
        println("Serie de Fibonacci es:")
        fibonacci(n)
    }
}

File: pregunta 1/estructurado/uno.py
def fibonacci(n):
    a=1
    b=0
    for i in range(n):
        print(b)
        b = a+b
        a = b-a
        i += 1


print("Ingrese un valor")
n = int(input())
print("La Serie de Fibonacci es: ")
fibonacci(n)